apps/api/src/test-data/buildDigitTextTestData.ts

```diff
@@ -0,0 +1,80 @@
+import { LanguageCode, MultilingualTextItemRole } from '@coscrad/api-interfaces';
+import { buildMultilingualTextFromBilingualText } from '../domain/common/build-multilingual-text-from-bilingual-text';
+import { buildMultilingualTextWithSingleItem } from '../domain/common/build-multilingual-text-with-single-item';
+import buildDummyUuid from '../domain/models/__tests__/utilities/buildDummyUuid';
+import { CreateDigitalText, DigitalTextCreated } from '../domain/models/digital-text/commands';
+import { DigitalText } from '../domain/models/digital-text/digital-text.entity';
+import { ResourceType } from '../domain/types/ResourceType';
+import { clonePlainObjectWithOverrides } from '../lib/utilities/clonePlainObjectWithOverrides';
+import { DTO } from '../types/DTO';
+import {
+    convertAggregatesIdToUuid,
+    convertSequenceNumberToUuid,
+} from './utilities/convertSequentialIdToUuid';
+
+const dummyDateNow = 1664237194999;
+
+const digitTextDtos: DTO<Omit<DigitalText, 'id'>>[] = [
+    {
+        type: ResourceType.digitalText,
+        title: buildMultilingualTextFromBilingualText(
+            {
+                text: 'Court Case Digital Text title in language',
+                languageCode: LanguageCode.Haida,
+            },
+            {
+                text: 'Court Case Digital Text title (English)',
+                languageCode: LanguageCode.English,
+            }
+        ),
+        published: true,
+    },
+    {
+        type: ResourceType.digitalText,
+        title: buildMultilingualTextWithSingleItem('Digital Text number two title'),
+        published: false,
+        queryAccessControlList: {
+            allowedUserIds: ['1'],
+            allowedGroupIds: [],
+        },
+    },
+];
+
+const createDigitTextCommands: CreateDigitalText[] = digitTextDtos.map(({ title }, index) => ({
```

Comment:
It's a shame we have to do this to get the event history. We need to phase out `buildXTestData`. On the next resource we add, we should avoid this test helper at all costs. We should op to use `seedInitialState` and use test command FSAs with overrides to do so in the command tests going forward.